cms/jinja2/templates/pages/errors/500.html

```diff
@@ -25,8 +25,8 @@
         </style>
     </head>
     <body>
-        <h1>Something went wrong</h1>
-        <p>{{ _("We’re sorry! Our server is acting up right now. Please try again soon.") }}</p>
-        <p class="errorcode">{{ _("Internal Server Error (500)") }}</p>
+        <h1>Sorry, there's a problem with the service</h1>
```

Comment:
There are multiple strings that have not been marked for translation extraction.

Reply:
I see, thanks for clarifying. We probably need to fallback to checking domains if we can't detect.